src/ucs/sys/sock.c

```diff
@@ -267,6 +327,15 @@ ucs_status_t ucs_socket_recv(int fd, void *data, size_t length,
                               "recv", err_cb, err_cb_arg);
 }
 
+ucs_status_t ucs_socket_sendv_nb(int fd, struct iovec *iov, size_t iov_cnt,
+                                 size_t *length_p, ucs_socket_io_err_cb_t err_cb,
+                                 void *err_cb_arg)
+{
+    return ucs_socket_do_iov_nb(fd, iov, iov_cnt, length_p,
+                                (ucs_socket_iov_func_t)sendmsg,
```

Comment:
can we avoid the casting (ucs_socket_iov_func_t)?

Reply:
yes, we can